.github/workflows/build.yml

```
@@ -242,12 +241,12 @@ jobs:
       CCACHE_NOCOMPRESS: 1
       MAKEFLAGS: -j2
     steps:
-      - name: Enable POSIX MinGW
+      - name: Configure apt
         run: |
-          update-alternatives --set i686-w64-mingw32-gcc /usr/bin/i686-w64-mingw32-gcc-posix
-          update-alternatives --set i686-w64-mingw32-g++ /usr/bin/i686-w64-mingw32-g++-posix
-          update-alternatives --set x86_64-w64-mingw32-gcc /usr/bin/x86_64-w64-mingw32-gcc-posix
-          update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
+          sudo add-apt-repository ppa:tobydox/mingw-w64
+          sudo sed -i -e "s/$(lsb_release -cs)/focal/g" \
+          /etc/apt/sources.list.d/tobydox-ubuntu-mingw-w64-$(lsb_release -cs).sources
+          sudo apt-get update -y
```

Could you see if this works?
```suggestion
          sudo sh -c 'echo "deb http://ppa.launchpad.net/tobydox/mingw-w64/ubuntu focal main" > /etc/apt/sources.list.d/tobydox-mingw-w64.list'
          sudo gpg --keyserver keyserver.ubuntu.com --recv-keys 72931B477E22FEFD47F8DECE02FE5F12ADDE29B2
          sudo apt-get update -y
```

This is based on [what the Docker image uses](https://github.com/LMMS/lmms-ci-docker/blob/master/linux.mingw/Dockerfile) and should work on both Ubuntu and Linux Mint.

---

Why would it need to change if our build runner changes? "focal" is the name of the Ubuntu version the PPA uses, not this build runner.

An alternative to `$(lsb_release -cs)` that works on both Ubuntu and Linux Mint is this:
```bash
. /etc/os-release
${UBUNTU_CODENAME}
```

---

> Why would it need to change if our build runner changes? "focal" is the name of the Ubuntu version the PPA uses, not this build runner.

It doesn't, I was confusing it with the winehq scripts in another PR that use a similar technique.  I'll switch it here and in the tutorial. 

With regards to the alternative approach, it's not in-line-able, so I don't think it works in CI, but is a nice workaround for the Linux wiki.

---

Build are failing due to the public key, which didn't occur with `add-apt-repository`.   This also means the [wiki changes](https://github.com/LMMS/lmms/wiki/Dependencies-Windows/_compare/5eeb4172858df831f1ba27094d8dbf806faf9356...018ab888a1c52f29f0280f603a3fb869fa0f04b4) need to be tweaked.

---

> Build are failing due to the public key

Fixed via
```bash
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 72931B477E22FEFD47F8DECE02FE5F12ADDE29B2
```

---

>  a nice workaround for the Linux wiki.

I double-checked and I don't use this `$(lsb_release -cs)` technique in the Ubuntu Linux wiki, so we should be good.